libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -493,6 +501,14 @@ const AP_Param::GroupInfo AP_InertialSensor::var_info[] = {
     // @Path: ../Filter/HarmonicNotchFilter.cpp
     AP_SUBGROUPINFO(_harmonic_notch_filter, "HNTCH_",  41, AP_InertialSensor, HarmonicNotchFilterParams),
 
+    // @Param: GYRO_RATE
+    // @DisplayName: Fast sampling gyro rate for enabled IMUs
```

Comment:
Maybe let's change this to "Gyro rate for IMUs with Fast Sampling enabled"

Reply:
Fixed